src/app/components/Image/index.tsx

```diff
@@ -165,9 +169,16 @@ const Image = ({
               loading={lazyLoad ? 'lazy' : undefined}
               width={width}
               height={height}
-              css={styles.image}
+              css={[
+                styles.image,
+                overridesAspectRatio && styles.imageAspectRatioOverride,
+              ]}
               fetchpriority={fetchpriority}
-              style={{ aspectRatio: `${aspectRatioX} / ${aspectRatioY}` }} // aspectRatio used in combination with the objectFit:cover will center the image horizontally and vertically if aspectRatio prop is different from image's intrinsic aspect ratio
+              style={{
+                aspectRatio: overridesAspectRatio
+                  ? 'auto'
+                  : `${aspectRatioX} / ${aspectRatioY}`,
+              }} // aspectRatio used in combination with the objectFit:cover will center the image horizontally and vertically if aspectRatio prop is different from image's intrinsic aspect ratio
```

Comment:
`<rant>` Not one for now, but I am not loving the combination of css & style here - I think it should be css. `</rant>`

Reply:
Yh I had a chat with Andrew and I'll make a tech ticket to look into this. 

The original motivation behind this is [documented in the readme](https://github.com/bbc/simorgh/blob/latest/src/app/components/Image/README.md#how-does-aspect-ratio-work). It's to reduce CSS bloat. (Note we wondered whether adding conditional styles like I have here could affect the page weight. But I compared the page weight of this to latest and there was no change.)

But it's possible there is a better/ different way of doing this so we can remove the inline styles. I'll make a tech ticket to look into this.